devices/device_config.bzl

```diff
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-def device_config(cpu, fpu, float_abi, endian):
+def device_config(cpu, arch, fpu, float_abi, endian):
```

Comment:
I'm not convinced that this is the best approach as arch and cpu should be mutually exclusive i.e. cortex-m0 has the architecture armv6-m. I don't want to be in a situation where there is a small mistake somewhere that might look like
``` python
cpu = "cortex-m0",
arch = "armv7-m",
```
As I am unsure how the compiler will behave when conflicting flags are set. So it should really be one or the other as there is otherwise redundant/conflicting information here.

Reply:
I didn't realize that. I will withdraw this change.